src/commands/create/make-docker-compose.js

```diff
@@ -89,6 +92,8 @@ module.exports = function makeDockerCompose( spinner ) {
 		if ( platform() == 'linux' ) {
 			baseConfig.services.phpfpm.image = `wp-php-fpm-dev-${ phpVersion }-${ slugify( process.env.USER ) }`;
 			baseConfig.services.phpfpm.volumes.push( `~/.ssh:/home/${ process.env.USER }/.ssh:cached` );
+			baseConfig.services.phpfpm.volumes.push( `${ wpsnapshotsDir }:/home/${ process.env.USER }/.wpsnapshots:cached` );
+			baseConfig.services.phpfpm.volumes.push( `~/.aws:/home/${ process.env.USER }/.aws:cached` );
```

Comment:
might be safer to mount ~/.aws as read only by adding :ro to the end of that `~/.aws:/home/www-data/.aws:cached:ro`.